server/src/server.ts

```diff
@@ -82,6 +84,29 @@ export class Server {
     return sock;
   }
 
+  private updateTemplate(template: Template): Template {
+    template.hash = crypto.createHash("sha256")
+      .update(JSON.stringify(template))
+      .digest("hex");
+
+    const cb0 = "03";                                       // push
```

Comment:
Minor: I think we should update the comment to say something like `// push length of the following height field` here.